src/svg/SvgImport.js

```diff
@@ -22,11 +22,12 @@ new function() {
     var definitions = {},
         rootSize;
 
-    function getValue(node, name, isString, allowNull, allowPercent) {
+    function getValue(node, name, isString, allowNull, allowPercent,
+                      defaultValue) {
```

Comment:
Regarding the 80 chars per line rule: I used to indent it like this, with 2 tabs (= 8 spaces) on the 2nd line, so it doesn't align with the code-block underneath and is easier to catch visually:

```js
    function getValue(node, name, isString, allowNull, allowPercent,
            defaultValue) {
```